src/utils/json.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function deepEqual(a: any, b: any): boolean {
```

Comment:
Same goes here as on the backend you can use generic types like `export function deepEqual<T>(a: T, b: T): boolean {` instead of using `any` and disabling the eslint rule.